lib/scsi_defs.h

```diff
@@ -1,10 +1,18 @@
 #ifndef LIB_SCSI_DEFS_H
 #define LIB_SCSI_DEFS_H
 
+#include <vector>
 #include <cstdint>
 
+#include "absl/base/attributes.h"
+
 namespace scsi_defs {
 
+  const uint32_t LUN_DATA_HEADER_SIZE = 8;
+  const uint32_t LUN_BYTE_SIZE = 8;
```

Comment:
We can use a generic name for this since this will be re-usable in other structures in this file as well

Reply:
Could you elaborate a bit on this?

Reply:
I'm using the same variable [here](https://github.com/googleinterns/scsi2nvme/pull/20/files#diff-74826f7cf214dcd85933fc081f197fbcR10) So, we can just rename it to something generic and not `LUN` specific

Reply:
Gotcha. I'm primarily using this variable to track the number of bytes in a LUN address. To allow for more dynamic code I'll remove this constant and use sizeof(LunAddress) instead.